src/browser/cached_resource_fetcher.ts

```diff
@@ -213,10 +228,8 @@ function download(fileUrl: string, filePath: string): Promise<any> {
             });
         });
 
-        request.once('login', (authInfo: any, callback: Function) => {
-            //Simply provide empty credentials to raise the authentication error.
-            //https://github.com/rdepena/electron/blob/master/docs/api/client-request.md#event-login
-            callback();
+        request.on('login', (authInfo: any, callback: Function) => {
```

Comment:
Should we clean up on the 'error' event like [here](https://github.com/HadoukenIO/core/pull/280/files#diff-f2ab184e212da874890dc8d44365e5bcR321)?